KsApi/MockService.swift

```diff
@@ -1277,6 +1277,10 @@
         .fetchWithResult(query: fetchProjectCommentsQuery, result: projectOnlyResult)
     }
 
+    internal func fetchProject_combine(project _: Project) -> AnyPublisher<Project, ErrorEnvelope> {
+      fatalError("Sorry, this is unimplemented!")
```

Comment:
This test method will never be used, so it's just here for protocol conformance.